### ISSUE TITLE
Specify the onfoo event listener attributes in Web IDL

### ISSUE THREAD

**Issue by foolip** [w3c/deviceorientation#17]
ondeviceorientation, oncompassneedscalibration and ondevicemotion are mentioned, but not spec'd in the way that most specs do.

Example: https://fullscreen.spec.whatwg.org/#api

**Comment by foolip** [comment]
@timvolodine, are you the main editor of the spec these days? I have a few issues, including this one, that have gotten no response since I filed them.

**Comment by timvolodine** [comment]
foolip@: is this issue still relevant?

**Comment by foolip** [comment]
Yes it is. The spec says "User agents must also provide an event handler IDL attribute named ondeviceorientation on the window object. The type of the corresponding event must be deviceorientation." which implementors will surely understand, but it can be done using IDL as in Blink:
```WebIDL
partial interface Window {
    attribute EventHandler ondeviceorientation;
    attribute EventHandler ondeviceorientationabsolute;
};
```
Note also that "The type of the corresponding event must be deviceorientation" sounds a bit strange, it should be something involving "event handler event types".

**Comment by timvolodine** [comment]
this should be addressed by the pull request #31 above, which is merged now

**Comment by foolip** [comment]
Changes in #31 LGTM
